packages/salesforcedx-vscode-lightning/src/telemetry/telemetry.ts

```diff
@@ -44,14 +74,25 @@ export class TelemetryService {
     }
   }
 
-  public sendExtensionDeactivationEvent(): void {
+  public async sendExtensionDeactivationEvent(): Promise<void> {
+    await this.setupVSCodeTelemetry();
     if (this.reporter !== undefined && this.isTelemetryEnabled) {
       this.reporter.sendTelemetryEvent('deactivationEvent', {
         extensionName: EXTENSION_NAME
       });
     }
   }
 
+  public async sendCommandEvent(commandName?: string): Promise<void> {
```

Comment:
This method is never used by the `salesforcedx-vscode-lightning` extension, you can remove it.